_issues/defending_child_abuser.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: post
+title: "Didn't she get a rapist acquitted, and then laugh about the girl who was raped?"
+categories: controversy
+---
+
+**Q:** Didn't Hillary get a rapist acquitted, and then laugh about the girl who was raped?  
+
+**A**: In the second Presidential debate, Donald Trump claimed "Her client she represented
+got him off and she is seen laughing on two occasions laughing at the girl who was raped." This accusation [has been repeatedly disproven](http://www.politico.com/blogs/2016-presidential-debate-fact-check/2016/10/trump-is-wrong-hillary-clinton-did-not-laugh-about-the-rape-of-a-12-year-old-229455#ixzz4MiTn0Cwt). Here are the facts:
+
+* In 1975, Hillary was **the court-appointed attorney** for a man being tried for the rape of a 12 year old girl. 
+* The defendant [specifically requested a female attorney](http://politicalticker.blogs.cnn.com/2014/06/25/prosecutor-in-controversial-case-says-clinton-had-no-choice-but-to-defend-rapist/comment-page-1/). 
+* Hillary was a young lawyer, only 27 at the time, and **did not want to take the case**. [According to the prosecutor, Mahlon Gibson](http://politicalticker.blogs.cnn.com/2014/06/25/prosecutor-in-controversial-case-says-clinton-had-no-choice-but-to-defend-rapist/comment-page-1/), Hillary approached him for advice:
+
+> {Hillary said} 'I don't want to represent this guy. I just can't stand this. I don't want to get involved. Can you get me off?' I told her, ‘Well contact the judge and see what he says about it,’ but I also said 'don't jump on him and make him mad.' She contacted the judge and the judge didn't remove her and she stayed on the case.
+
+* The [Sixth Amendment of the U.S. Constitution](http://constitution.findlaw.com/amendment6.html) guarantees the right to an attorney. As a result, Hillary was **obligated to defend her client to the best of her ability**, which she did. According to Gibson:
+
+> I would say she did a heck of a job of representing her client to the best of her ability. If you are appointed by the court, you damn better do everything you can do to defend them.
+
+* The case resulted in a plea bargain, in which **the defendant was convicted** of a lesser charge.
+
+**Q**: But how could Hillary laugh at a girl who was raped?
+
+**A**: She didn't. This accusation stems from <iframe width="560" height="315" src="https://www.youtube.com/watch?v=e2f13f2awK4" frameborder="0" allowfullscreen>a recorded interview with an Arkansas journalist</iframe> in the 1980's. In the **six minute recording**, Hillary can be heard chuckling four times. However, the points she was laughing about had [nothing to do with the central points of the case, and she **certainly was not laughing at the victim**](http://www.politico.com/blogs/2016-presidential-debate-fact-check/2016/10/trump-is-wrong-hillary-clinton-did-not-laugh-about-the-rape-of-a-12-year-old-229455#ixzz4MiTn0Cwt).
```

Comment:
Ah I see. Your use of the `iframe` here is not quite right. `iframe` is used to embed the video itself (and the video player) into the article. So in that case you'd want it on its own paragraph with no text between the opening `<iframe ...>` tag and the closing `</iframe>`.

If you simply want to _link_ to a video, you can use the `[simple link notation](https://www.youtube.com/...)` you ahve been using before.